web/admin.py

```
@@ -7,7 +7,8 @@
 from django.http import HttpResponseRedirect
 from django.urls import path, reverse
 from django.utils.html import format_html
-
+from django.core.exceptions import ValidationError
```

Ruff (0.8.2)

`django.core.exceptions.ValidationError` imported but unused

Remove unused import: `django.core.exceptions.ValidationError`

(F401)